api/handler.js

```diff
@@ -1,6 +1,6 @@
 'use strict'; // eslint-disable-line strict
 
-const handle = require('./graphql/index').default;
+const handle = require('../graphql/index').default;
```

Comment:
This should be `./graphql/index` since it's the code compiled from `graphql` to `api/graphql`